src/imageops/affine.rs

```diff
@@ -97,3 +97,115 @@ pub fn flip_vertical<I: GenericImage + 'static>(image:  &I)
 
     out
 }
+
+#[cfg(test)]
+mod test {
+    use super::{rotate90,rotate180,rotate270,flip_horizontal,flip_vertical};
+    use buffer::{ImageBuffer,Pixel,GrayImage};
+    use image::{GenericImage};
+
+    macro_rules! assert_pixels_eq {
+        ($actual:expr, $expected:expr) => ({
+            let actual_dim = $actual.dimensions();
+            let expected_dim = $expected.dimensions();
+
+            if actual_dim != expected_dim {
+                panic!("dimensions do not match. \
+                    actual: {:?}, expected: {:?}", actual_dim, expected_dim)
+            }
+
+            let diffs = pixel_diffs($actual, $expected);
+
+            if !diffs.is_empty() {
+                let mut err = "pixels do not match. ".to_string();
+
+                let diff_messages = diffs
+                    .iter()
```

Comment:
As the iterator doesn't need ownership of the diffs I just used the regular iter(). It seems reasonable to me to use the least demanding iterator that works, but I'll change if anyone objects.